examples/contentful-simple/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "icons": "npx @svgr/cli --typescript --replace-attr-values \"#000=currentColor\" --out-dir generated/icons -- public/icons",
+    "codegen-ts": "graphql-codegen --config graphql.config.ts --watch"
+  },
+  "dependencies": {
+    "@conform-to/react": "^1.6.1",
+    "@conform-to/zod": "^1.6.1",
+    "@contentful/rich-text-html-renderer": "^17.0.0",
+    "@contentful/rich-text-react-renderer": "^16.0.1",
+    "@contentful/rich-text-types": "^17.0.1",
+    "@graphql-codegen/cli": "^5.0.6",
```

Comment:
We should move `@graphql-codegen/cli`, `@parcel/watcher`, `postcss`, and `tailwindcss` to `devDependencies`.